src/app/rundown-view/components/rundown-header-context-menu/rundown-header-context-menu.component.ts

```diff
@@ -48,8 +52,23 @@ export class RundownHeaderContextMenuComponent {
     },
   ]
 
+  private readonly contextMenuOptionsForRehearseRundown: ContextMenuOption[] = [
+    {
+      label: 'Activate (On Air)',
+      contextAction: (): void => this.openActivateRundownDialog(),
+    },
+    ...this.contextMenuOptionsForActiveRundown,
```

Comment:
This makes use of `resetRundown()` which does not consider the fact that we might be in Rehearsal prior to a reset. So we will have a Rehearsal rundown, then reset and end up with an Activated rundown instead.

Reply:
@RasmusAlbrektsen We check that, but that bug is related to back-end code, no front-end. After Reset rundown back-end set rundown to deactivate and after that push it to active again, front-end just fetch again the rundown data and that is the reason to see it in front-end too. That is discuss it with @LindvedKrvang  and he will make a back-end task for this issue.